### PR TITLE
creates script to run monthly stats email task

### PR DESCRIPTION
**Pull request by whereismyjetpack**
Kubernetes CronJob runs `/app/bin/monthly_stats_mail`, on a configurable schedule.

### CODE REVIEW

File: bin/envconsul-wrapper [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    export VAULT_TOKEN=$vault_token
+fi
+
+function start() {
+    if [ "$1" != "" ]; then
+        cmd="$*"
+        if [ -n "${VAULT_TOKEN}" ]; then
+            echo "have token. starting envconsul"
+            start_envconsul
+        else
```
Comment (awead) [inline bin/envconsul-wrapper:22]: This implies that the command can still run if there is no vault token present. Is this a valid scenario?

File: config/sidekiq.yml [hunk 2]
```diff
@@ -2,4 +2,5 @@
 :max_retries: <%= ENV.fetch("SIDEKIQ_MAX_RETRIES", 2) %>
 :queues:
   - shrine
+  - mailers
```
Comment (awead) [inline config/sidekiq.yml:5]: Do we need this?

**Comment by whereismyjetpack** [comment]
Totally agree. I think we can have the container setup the env and then maybe use positional arguments for the commands. That would require a bit more reworking of things but I'll certainly think on it

File: config/sidekiq.yml (hunk 2 above)
Comment (whereismyjetpack) [reply config/sidekiq.yml:5]: Acionmailer dropped the messages on the mailers queue. I needed to setup sidekiq to read it in

File: bin/envconsul-wrapper [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+if [ -f /vault/token ]; then
+    vault_token=$(cat /vault/token)
+    export VAULT_TOKEN=$vault_token
+fi
+
+# New versions of vault-init calls the file something slightly different. export vault token if that file is found
+if [ -f /vault/.vault-token ]; then
+    vault_token=$(cat /vault/.vault-token)
+    export VAULT_TOKEN=$vault_token
+fi
```
[the remark this answers is not in the document]
Comment (whereismyjetpack) [reply bin/envconsul-wrapper:14]: Good catch.

File: bin/envconsul-wrapper (hunk 1 above)
Comment (whereismyjetpack) [reply bin/envconsul-wrapper:22]: The command can still run if the env is setup outside of vault. For testing etc.

File: bin/envconsul-wrapper (hunk 3 above)
[the remark this answers is not in the document]
Comment (whereismyjetpack) [reply bin/envconsul-wrapper:8]: `/vault/token` is no longer used, so i just removed that branch.

**Pull request merged**